files/zh-cn/web/api/node/index.md

```diff
@@ -227,7 +225,7 @@ eachNode(box, function(node){
 用户终端上会显示如下字符：
 
 ```js
-"\n\t", "Foo", "\n\t", "Bar", "\n\t", "Baz"
+"\n\t", "Foo", "\n\t", "Bar", "\n\t", "Baz";
```

Comment:
This should have been nolint, however, the whole example was not in en-US document, so we can safely ignore this :(